internal/core/src/segcore/SegmentSealedImpl.cpp

```diff
@@ -1514,14 +1544,17 @@ SegmentSealedImpl::generate_binlog_index(const FieldId field_id) {
     auto& field_index_meta = col_index_meta_->GetFieldIndexMeta(field_id);
     auto& index_params = field_index_meta.GetIndexParams();
 
+    bool is_sparse =
```

Comment:
generate_binlog_index should rename to generate interim index.

Reply:
done